sensor_msgs/Image map
---
std_msgs/String answer
